config.yaml

```
@@ -1,6 +1,6 @@
 options:
   channel:
     type: string
-    default: "1.29/stable"
+    default: "1.29/beta"
     description: |
-      Snap channel to install Kubernetes snaps from
+      Snap channel to install Kubernetes snaps from
```

We lost a CR here at the end of this file.  Also, this description can use some clean up
```suggestion
      Snap channel from which to install the kubernetes-test snap
```